src/main/java/it/lucchesi/tools/DateTools/DateTools.java

```diff
@@ -26,6 +26,8 @@ public static long weekdays(LocalDate start, LocalDate end) {
 		Preconditions.checkArgument(Preconditions.checkNotNull(start)
 				.compareTo(Preconditions.checkNotNull(end)) <= 0);
 		
+		if(start.equals(end)) return 1;
```

Comment:
You're right, the test was the first thing I did after I received your pull request, it's on the master branch.